src/xmlgen.cpp

```diff
@@ -132,6 +132,7 @@ inline void writeXMLCodeString(FTextStream &t,const char *s, int &col)
       case '&':  t << "&amp;"; col++;  break;
       case '\'': t << "&apos;"; col++; break;
       case '"':  t << "&quot;"; col++; break;
+      case 0xA9:  t << "&#169;"; col++; break;
```

Comment:
This change is not needed in my opinion. The XML output is already UTF-8 encoded, so any non-ascii character should appear as such, and are handled by the default case.